manager/manager.go

```diff
@@ -126,6 +126,12 @@ type Config struct {
 	// first node in the cluster, this setting is used to set the cluster-wide mandatory
 	// FIPS setting.
 	FIPS bool
+
```

Comment:
@dperny this does not look like the right place for this, is there any other better place for the network config?
At least seems like a good idea to condense the 2 into a single structure, right now they are spread everywhere

Reply:
I agree. I think we should create NetworkConfig in manager/controlapi/network.go and add these fields to it @selansen

Reply:
I would prefer this go in with this PR, but if we're time constrained, we can do the refactor as a follow-up refactor. @thaJeztah @selansen